service/controller/chart/v1/resource_set.go

```diff
@@ -146,6 +162,7 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 		chartMigrationResource,
 		tillerResource,
 		tillerMigrationResource,
+		releaseMigrationResource,
```

Comment:
Running the resource just after `tillermigration`.

Reply:
I think it's safest to execute `releaseMigration` first. If the 2-to-3 plugin is not done I don't think we should do any of the other steps. WDYT?

Reply:
Sure, What I was worried about is e2etest failure, but I can ignore that.